### PR TITLE
[qa] pull-tester: Don't mute zmq ImportError

### PR DESCRIPTION
**Pull request by maflcko**
Let's consider muting exceptions bad practice. It is good to have a more verbose output, so I left this as is.

### CODE REVIEW

**Comment by MarcoFalke** [comment]
Looks like I need to move some code; use https://github.com/bitcoin/bitcoin/commit/faa4f22?w=0 to check the second diff.

File: qa/pull-tester/rpc-tests.py [hunk 1]
```diff
@@ -76,11 +67,24 @@
 if "BITCOINCLI" not in os.environ:
     os.environ["BITCOINCLI"] = buildDir + '/src/bitcoin-cli' + EXEEXT
 
-#Disable Windows tests by default
 if EXEEXT == ".exe" and "-win" not in opts:
-    print "Win tests currently disabled.  Use -win option to enable"
+    # https://github.com/bitcoin/bitcoin/commit/d52802551752140cf41f0d9a225a43e84404d3e9
```
Comment (laanwj) [inline qa/pull-tester/rpc-tests.py]: Makes sense to refer to the discussion of the pull this was part of: https://github.com/bitcoin/bitcoin/pull/5677#issuecomment-136646964

**Comment by laanwj** [comment]
Concept ACK

File: qa/pull-tester/rpc-tests.py (hunk 1 above)
Comment (MarcoFalke) [inline qa/pull-tester/rpc-tests.py]: Done

File: qa/pull-tester/rpc-tests.py [hunk 2]
```diff
@@ -76,11 +67,25 @@
...
+if not (ENABLE_WALLET == 1 and ENABLE_UTILS == 1 and ENABLE_BITCOIND == 1):
+    print "No rpc tests to run. Wallet, utils, and bitcoind must all be enabled"
     sys.exit(0)
 
+# python-zmq may not be installed. Handle this gracefully and with some helpful info
+if ENABLE_ZMQ:
+    try:
+        import zmq
+    except ImportError as e:
+        print("WARNING: \"import zmq\" failed. Set ENABLE_ZMQ=0 or " \
```
Comment (laanwj) [inline qa/pull-tester/rpc-tests.py]: This is no longer a WARNING but an ERROR now

Comment (MarcoFalke) [inline qa/pull-tester/rpc-tests.py]: Done

**Pull request merged**